lib/tasks/reminder.rake

```diff
@@ -0,0 +1,24 @@
+namespace :reminder do
+  desc "Send actionable approver reminder email for pending requests"
+  task actionable_approvers: :environment do
+    proposals = Proposal.pending.where(client_data_type: "Ncr::WorkOrder")
+    proposals.each do |proposal|
+      send_reminder_email(proposal)
+    end
+  end
+
+  private
+
+  def send_reminder_email(proposal)
+    user = proposal.client_data.current_approver
+    unless user
```

Comment:
Take it up with rubocop. That's why I changed it.

I don't disagree with you; just requires a change to our rules. Maybe start a VOTE thread on slack or a PR to the `.rubocop.yml` ?